Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for SUSE Cloud Foundry
+name: cf
```

Comment:
There's more metadata we can add here.
Can you ping @troytop to give us some values?